comparadise-utils/match-screenshot.ts

```diff
@@ -72,16 +74,20 @@ function verifyImages() {
   }
 }
 
+interface SensitivityOptions {
```

Comment:
Super nit pick, but I prefer using `type` over `interface` since the only reason to use `interface` is if you are going to inherit from it.
```suggestion
type SensitivityOptions = {
```

You could even just inline this type and not give it a name if you want